applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_analysis.py

```diff
@@ -138,6 +137,9 @@ def __ExecuteInitialize(self):
         ## Adds the Dofs if they don't exist
         self.solver.AddDofs()
 
+        # Initialize IO
```

Comment:
Why did you have to move this here?
I would prefer to not nest the functions too much

Reply:
It is because I needed for my adaptative strategy in order to post-process inner steps